src/gitHubTypings.ts

```diff
@@ -3,12 +3,14 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-import { ActionsGetJobForWorkflowRunResponseData, GitGetTreeResponseData, OrgsListForAuthenticatedUserResponseData } from "@octokit/types";
+import { ActionsGetJobForWorkflowRunResponseData, GitGetTreeResponseData, OrgsListForAuthenticatedUserResponseData, ReposListBranchesResponseData, ReposListForOrgResponseData } from "@octokit/types";
 
 // we'll export the types until this is fixed: https://github.com/octokit/types.ts/issues/120
 export type OrgForAuthenticatedUserData = OrgsListForAuthenticatedUserResponseData[0];
 export type GitTreeData = GitGetTreeResponseData['tree'][0];
 export type ActionWorkflowStepData = ActionsGetJobForWorkflowRunResponseData['steps'][0];
+export type RepoData = ReposListForOrgResponseData[0];
+export type BranchData = ReposListBranchesResponseData[0];
```

Comment:
Is the zero index on these types just a way of converting an array to a single element of that type? I haven't seen that pattern before

Reply:
Yeah... this is a weird thing.

![image](https://user-images.githubusercontent.com/5290572/94867841-d8426480-03f6-11eb-8d54-672f905357d8.png)
![image](https://user-images.githubusercontent.com/5290572/94867831-d37db080-03f6-11eb-8ede-22c2f0e9210b.png)

Notice that the typing ends with a [].  Dunno _why_ that they did this, but basically the data type that I want to extract lives in the 0 index.  Rather than use `ReposListBranchesResponseData[0]`, I just extracted the type that I wanted and renamed it to what it is (which is just RepoData)